activerecord/test/models/cpk/comment.rb

```diff
@@ -3,7 +3,7 @@
 module Cpk
   class Comment < ActiveRecord::Base
     self.table_name = :cpk_comments
-    belongs_to :commentable, class_name: "Cpk::Post", foreign_key: %i[commentable_title commentable_author], polymorphic: true
-    belongs_to :post, class_name: "Cpk::Post", foreign_key: %i[commentable_title commentable_author]
+    belongs_to :commentable, foreign_key: %i[commentable_title commentable_author], polymorphic: true
+    belongs_to :post, foreign_key: %i[commentable_title commentable_author]
```

Comment:
Hey @fxn, I was curious why `class_name` was removed from `belong_to :post` since it doesn't have `polymorphic: true` (I tried locally and it didn't raise or warn when present).

Reply:
@skipkayhil Right! I just restored it in https://github.com/rails/rails/commit/612666219b8c44770479c366d9efc936269033ee. Thanks!